src/localization_tf_plugin.cpp

```diff
@@ -34,32 +27,29 @@ LocalizationTFPlugin::~LocalizationTFPlugin()
 void LocalizationTFPlugin::configure(tue::Configuration config)
 {
     config.value("robot_name", robot_name_);
-
```

Comment:
Was there a specific reason here why the listener was deleted and restarted?

Reply:
That did cross my mind too. But I couldn't think of a situation in which restarting would make a difference.